javaparser-core/src/main/java/com/github/javaparser/JavaParser.java

```diff
@@ -45,15 +45,19 @@
 import static com.github.javaparser.ParseStart.*;
 import static com.github.javaparser.Problem.PROBLEM_BY_BEGIN_POSITION;
 import static com.github.javaparser.Providers.*;
-import static com.github.javaparser.utils.Utils.assertNotNull;
+import static com.github.javaparser.utils.Utils.assertNotNull;import org.apache.log4j.Logger;
+
```

Comment:
javaparser-core has no dependencies and it should stay that way. So no log4j. And even then there would have been a preference for slf4j.